test/PokemonPage.jest.spec.jsx

```diff
@@ -93,12 +93,12 @@ describe('<PokemonPage />', () => {
   })
 
   it('should render stats', async () => {
-    axiosMock.get.mockResolvedValueOnce({ data: pokemon })
+    axiosMock.get.mockResolvedValueOnce({ data: pokemonList })
 
     await act(async () => {
       render(
         <Router history={history}>
-          <PokemonPage pokemons={pokemons} />
+          <PokemonPage pokemonList={pokemons} />
```

Comment:
The value of the prop is still pokemons 🙈

Reply:
Not anymore 🙃